app/i18n/de.json

```diff
@@ -23,7 +23,8 @@
       "OVERVIEW": "Übersicht",
       "CONSOLE": "Konsole",
       "TABLE": "Tabelle",
-      "CLUSTER": "Cluster"
+      "CLUSTER": "Cluster",
+      "GET_STARTED":"Loslegen"
```

Comment:
Is it somehow possible that the tutorial plugin provides its own i18n json file?
Not sure whether it is really doable, though.